apps/hostmetrics.go

```diff
@@ -76,6 +313,7 @@ func (r MetricsReceiverHostmetrics) Pipelines(ctx context.Context) []otel.Receiv
 					},
 				},
 			},
+			//transforms...,
```

Comment:
Is this left over from earlier debugging?

Reply:
Whoops, thanks for the catch.